src/website/app/Page.js

```diff
@@ -244,13 +244,15 @@ const styles = {
         paddingTop: pxToEm(53), // roughly theme.baseline_5
 
         '&::before': {
+          content: '""',
```

Comment:
Header in this file is a styled Section, which already has this property for these selectors: https://github.com/mineral-ui/mineral-ui/blob/master/src/website/app/Section.js#L59

Reply:
Removed... Must've added when trying to get site to render initially.